### PR TITLE
added relations for the SIR model

### PR DESCRIPTION
**Pull request by infvie**
- also added edge-vertex by accident. should be able to power thorough the rest much faster. 
I had a realization that the text it self very much assumes alot of basic knowledge that a program may have a hard time distinguishing. An example of this would be that susceptible people become infected and then move to recovered. It isn't explicitly stated which might make these difficult to reason.

### CODE REVIEW

**Comment by jpfairbanks** [comment]
> An example of this would be that susceptible people become infected and then move to recovered. It isn't explicitly stated which might make these difficult to reason.

Yeah this is information we need to get at by reading the wikipedia / using an ontology like framenet or the unified verb index. Getting it out manually by reading the text and making edges is a good start, (that is how ontologies start).

File: examples/knowledge_graph/sree/relations.csv [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+http://epirecip.es/epicookbook/chapters/sir/intro,"The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick as a special case of a more general model, and forms the framework",susceptible-infected-recovered (SIR) model,framework,forms
+http://epirecip.es/epicookbook/chapters/sir/intro,"The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick as a special case of a more general model, and forms the framework of many compartmental models. ",susceptible-infected-recovered (SIR) model,compartmental models,forms
+http://epirecip.es/epicookbook/chapters/sir/intro,"Susceptible individuals, S, are infected by infected individuals, I, at a per-capita rate βI, and infected individuals recover at a per-capita rate γ to become recovered individuals, R.",susceptible ,infected ,infeced by
+http://epirecip.es/epicookbook/chapters/sir/intro,"Susceptible individuals, S, are infected by infected individuals, I, at a per-capita rate βI, and infected individuals recover at a per-capita rate γ to become recovered individuals, R.",infected,infect,at
```
Comment (jpfairbanks) [inline examples/knowledge_graph/sree/relations.csv]: Should probably be something like: `susceptible individuals, infected individuals, are infected by`

**Comment by jpfairbanks** [comment]
lets move this folder from `sree` to `manual`

File: examples/knowledge_graph/sree/relations.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+pageURL,originaltext,subject,object ,verb
+http://epirecip.es/epicookbook/chapters/sir/intro,The susceptible-infected-recovered (SIR) model in a closed population,susceptible-infected-recovered (SIR) model,closed population,in
+http://epirecip.es/epicookbook/chapters/sir/intro,The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick,susceptible-infected-recovered (SIR) model,Kermack and McKendrick,proposed
+http://epirecip.es/epicookbook/chapters/sir/intro,The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick as a special case,susceptible-infected-recovered (SIR) model,special case,as
```
[the remark this answers is not in the document]
Comment (jpfairbanks) [reply examples/knowledge_graph/sree/relations.csv]: That is a good question Scott. I think we need to do some information extraction from the text in order to make human readable explanations and presentations of the models. We also need to connect what will be terse and generic code written by the scientists into a coherent framework for reasoning. An example of this is when `S` appears in one model (which we get from parsing/running the code) how do we know that it is referring to the *number of susceptibles* and, that number is the same as a variable called `sus_count` from another model?

If we want to link models into metamodels, we will need some common ground knowledge to make these joins on. We know that "same name, same type, same unit" is not enough for meaningful joins on programs. 

What do you have in mind for how we can use information from the text for the other

[the remark this answers is not in the document]
Comment (scottagt) [reply examples/knowledge_graph/sree/relations.csv]: > What do you have in mind for how we can use information from the text for the other
Sorry, what's "other" refer to? Or did the sentence cut-off before comment submission?

In any case, I think its useful to continue to flesh out the use cases (referencing #23) that we expect and plan for users to be able to work with when using the system. I think knowing what the elements of those use cases are will drive the discovery/development of specific kinds of meta-modeling functionalities. And then knowing what these functionalities are going to be will motivate the selection of particular kinds of knowledge that would need be present in a code/text KGs derived from individual researcher's prior scripts associated with their research publications to be used in meta-modeling. The evidence/reason for selecting to include particular kinds of attributes in our info extraction is that it's being planned for use in a use case for a specific goal, (and not just being made generally available without a particular reason).

**Comment by jpfairbanks** [comment]
@infvie please do manual annotation of the following files first. We need the association over every variable and function with the concept that it represents.

examples/epicookbook/src/
0. ScalingModel.jl
1. SEIRmodel.jl
2. SIRModel.jl
3. SemiParamSIRJulia.jl
4. SISMode.jl

**Pull request merged**